src/main/java/com/microsoft/sqlserver/jdbc/SQLServerDriver.java

```diff
@@ -1251,6 +1266,7 @@ public java.sql.Connection connect(String url, Properties suppliedProperties) th
                     "Microsoft JDBC Driver " + SQLJdbcVersion.MAJOR + "." + SQLJdbcVersion.MINOR + "."
                             + SQLJdbcVersion.PATCH + "." + SQLJdbcVersion.BUILD + SQLJdbcVersion.RELEASE_EXT
                             + " for SQL Server");
+            logClientOSAndArchInfo();
```

Comment:
is this meant to be used elsewhere? if only used once here then there's no need to create a sep method